ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/LLMs/422__model_serving_llm_models.robot

```diff
@@ -0,0 +1,72 @@
+*** Settings ***
+Documentation     Collection of CLI tests to validate the model serving stack for Large Language Models (LLM).
+...               These tests leverage on TGIS Standalone Serving Runtime
+Resource          ../../../../Resources/OCP.resource
+Resource          ../../../../Resources/CLI/ModelServing/llm.resource
+Library            OpenShiftLibrary
+Suite Setup       Suite Setup
+Suite Teardown    RHOSi Teardown
+Test Tags         KServe
+
+
+*** Variables ***
+${TEST_NS}=    tgis-models
+${TGIS_RUNTIME_NAME}=    tgis-runtime
+${USE_PVC}=    ${TRUE}
+${DOWNLOAD_IN_PVC}=    ${TRUE}
+${USE_GPU}=    ${FALSE}
+
+
+*** Test Cases ***
+Verify User Can Serve And Query A bigscience/mt0-xxl Model
+    [Documentation]    Basic tests for preparing, deploying and querying a LLM model
+    ...                using Kserve and Caikit+TGIS runtime
+    [Tags]    Tier1
+    Setup Test Variables    model_name=mt0-xxl-hf    use_pvc=${USE_PVC}    use_gpu=${USE_GPU}
+    Set Project And Runtime    runtime=${TGIS_RUNTIME_NAME}     namespace=${test_namespace}
+    ...    download_in_pvc=${DOWNLOAD_IN_PVC}    model_name=${model_name}    
+    Compile Inference Service YAML    isvc_name=${model_name}
+    ...    sa_name=${DEFAULT_BUCKET_SA_NAME}
+    ...    model_storage_uri=${storage_uri}
+    ...    model_format=pytorch    serving_runtime=${TGIS_RUNTIME_NAME}
+    ...    limits_dict=${limits}
+    Deploy Model Via CLI    isvc_filepath=${INFERENCESERVICE_FILLED_FILEPATH}
+    ...    namespace=${test_namespace}
+    Wait For Pods To Be Ready    label_selector=serving.kserve.io/inferenceservice=${model_name}
+    ...    namespace=${test_namespace}
+    Query Model Multiple Times    model_name=${model_name}    runtime=${TGIS_RUNTIME_NAME}
+    ...    inference_type=all-tokens    n_times=1
+    ...    namespace=${test_namespace}    validate_response=${FALSE}    # temp
+    Query Model Multiple Times    model_name=${model_name}    runtime=${TGIS_RUNTIME_NAME}
+    ...    inference_type=streaming    n_times=1
+    ...    namespace=${test_namespace}    validate_response=${FALSE}
+    [Teardown]    Clean Up Test Project    test_ns=${test_namespace}
```

Comment:
We should add part to delete pvc as apart of cleanup

Reply:
deleting the NS the PVC gets deleted. But I can add the explicit deletion

Reply:
what i mean is if you delete the ns only pvc will be deleted but pv entry will still exits

Reply:
hm you sure? I will check

Reply:
pv is not ns scoped

Reply:
I double checked, the PV gets released and deleted automatically after deleting the PVC. It depends on the Storage class configuration: https://kubernetes.io/docs/concepts/storage/persistent-volumes/#delete . I think we can add the logic to delete the PV explicitly in another PR so we speed up now